arango_rdf/main.py

```diff
@@ -351,16 +358,14 @@ def export(self, file_name: str, format: str) -> None:
         # init rdf graph
         g = Graph()
 
-        # get all collections from db
-        collections = self.db.collections()
         names = []
-        for i in collections:
-            if i["name"][0] != "_":
-                names.append(i["name"])
+        col: StandardCollection
+        for _, col in self.collections.items():
```

Comment:
Not sure why, but the `export()` method was previously operating under the assumption that **all** ArangoDB collections were to be exported to an RDF file. 

The current change will instead export the following ArangoDB collections into RDF:
* IRI
* BNode
* Literal
* Statement

Reply:
Having them export into a single file result in proper RDF statements. Having separate files splits up the statements.

However, splitting based on graph name would be the most sensible thing. This is especially true for the Ontologies being exported as they aren't typically located in the same file as the RDF graphs/statements.

Reply:
Ah unless your comment is just stating it will no longer export additional collections than the ones you mentioned?

Reply:
> Ah unless your comment is just stating it will no longer export additional collections than the ones you mentioned?
yep exactly